app/lib/app/SecvisogramPage/shared/isPropertyRelevant.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {object} params
+ * @param {string[]} params.relevanceLevels
+ * @param {import("../../shared/types").DocumentCategory} params.category
+ * @param {import('./isPropertyRelevant/types').Property} params.property
+ * @param {string} params.selectedRelevanceLevel
+ */
+export default function ({
```

Comment:
this function would be a nice candidate for a small unit test :)

Reply:
I like the e2e test ;)